public/js/components/Tabs.js

```diff
@@ -10,12 +10,9 @@ const { debugTab } = require("../client");
 require("./Tabs.css");
 const dom = React.DOM;
 
-function Tabs({ tabs, newSource, paused, resumed,
-  selectTab, selectSource, loadSources }) {
-  function onClickTab(e) {
-    const tabActor = e.currentTarget.dataset.actorId;
-    debugTab({ tabActor, newSource, paused, resumed,
-               selectTab, selectSource, loadSources });
+function Tabs({ tabs, actions: boundActions }) {
+  function onSelect(tab) {
+    debugTab(tab.toJS(), boundActions);
```

Comment:
Ah good, I wasn't sure if you liked it the other way for clarity, but I think this works well!